src/engines/gstengine.cpp

```diff
@@ -860,7 +864,9 @@ shared_ptr<GstEnginePipeline> GstEngine::CreatePipeline(
       qLog(Error) << "Could not initialize pipeline" << kEnterprisePipeline;
       ret.reset();
     }
-  } else {
+  } else 
+#endif // HAVE_BACKGROUND_STREAMS
```

Comment:
Maybe cleaner to return in the two if blocks above and get rid of the else?

Reply:
Done.